docs/reference/settings.rst

```diff
@@ -315,6 +315,17 @@ Specifies the number of items per page shown when viewing an image's usage (see
 
 Specifies the number of images shown per page in the image chooser modal.
 
+``WAGTAILIMAGES_RENDITION_STORAGE``
+-----------------------------------
+
+If specified, this setting allows renditions to be stored using an alternate storage backend (default is None). This value should be a subclass of storages.backends. For example:
```

Comment:
would be good to be clear here, that the default is `None` which will use Django's default [`FileSystemStorage`](https://docs.djangoproject.com/en/4.0/ref/files/storage/#django.core.files.storage.FileSystemStorage).